lib/failure.sh

```diff
@@ -583,6 +683,12 @@ warn() {
   echo ""
 }
 
+warn_aws_proxy() {
+  if { [[ -n "$HTTP_PROXY" ]] || [[ -n "$HTTPS_PROXY" ]]; } && [[ "$NO_PROXY" != "amazonaws.com" ]]; then
+    warn "Your build may fail if NO_PROXY is not set to amazonaws.com" "https://devcenter.heroku.com/articles/troubleshooting-node-deploys#aws-proxy-error"
```

Comment:
We might want to duplicate this doc and change the link.

Reply:
It seems so specific that I wouldn't bother duplicating it... What do you think?

Reply:
Well i would love to not have any reference to heroku. But i understand that it could be annoying to duplicate it ... Your call on that.

Reply:
OK my call is to not duplicate this ^^